oic.r.csr.raml

```diff
@@ -0,0 +1,35 @@
+#%RAML 0.8
+title: OICSecurityCsrResource
+version: v1.0-20150819
+schemas:
+  - Csr: !include schemas/oic.r.csr.json
+
+traits:
+  - interface:
+      queryParameters:
+        if:
+            enum: ["oic.if.baseline"]
+
+/oic/sec/csr:
+  description: |
+    This resource specifies a Certificate Signing Request.
+
+  displayName: Certificate Signing Request
+  is: [ interface ]
+
+  get:
+    description: |
+      Retrieves CSR data.
+    responses:
+      200:
+        body:
+          application/json:
+            schema: Csr
+            example: |
+              {
+              "rt": ["oic.r.csr"],
+              "encoding" : "oic.sec.encoding.pem"
+              "csr": "PEMENCODEDCSR"
+              }
+      503:
```

Comment:
That's a sensible error code to return if the resource isn't present. Is that something we would specify here, though? It's not an error code returned if the resource is present and supported.